.flake8

```diff
@@ -0,0 +1,12 @@
+[flake8]
+exclude = 
+#    tests,
```

Comment:
yeah, handled by the next stanza down, where I use flake8-putty to ignore a subset of errors (mostly line length, and some style things about unused variables)